ods_ci/tests/Resources/Page/DistributedWorkloads/WorkloadMetricsUI.resource

```diff
@@ -7,7 +7,7 @@ Resource         ../../Common.robot
 
 *** Variables ***
 ${PROJECT_XP}=    xpath=//div[text()='Project']
-${DISTRIBUITED_WORKLOAD_METRICS_TITLE_XP}=    xpath=//h1[text()="Distributed Workload Metrics"]
+${DISTRIBUITED_WORKLOAD_METRICS_TITLE_XP}=    xpath=//h1[@data-testid="app-page-title"]
```

Comment:
This makes the verification too common, since very page has this `app-page-title` id.
Look at https://github.com/red-hat-data-services/ods-ci/blob/9d28a611a8466310ce58e1ba508ba49a71c806ec/ods_ci/tests/Tests/0600__distributed_workloads/test-distributed-workloads-metrics-ui.robot#L36
It tries to verify that we're in Distributed Workload Metrics. 

btw, maybe you can also test it with `Wait For Dashboard Page Title   Distributed Workload Metrics`

Reply:
@manosnoam Thanks for reviewing. Added `Wait For Dashboard Page Title   Distributed Workload Metrics` to verify